tests/integration/test_ses.snapshot.json

```diff
@@ -600,5 +600,101 @@
+          "Type": "Sender"
+        },
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 400
+        }
+      }
+    }
+  },
+  "tests/integration/test_ses.py::TestSES::test_invalid_tags_send_email[test_invalid_name:123-test_invalid_value:123]": {
+    "recorded-date": "06-07-2023, 17:23:15",
+    "recorded-content": {
+      "response": {
+        "Error": {
+          "Code": "InvalidParameterValue",
+          "Message": "Invalid tag name <test_invalid_name:123>: only alphanumeric ASCII characters, '_', and '-' are allowed.",
+          "Type": "Sender"
+        },
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 400
+        }
+      }
+    }
+  },
+  "tests/integration/test_ses.py::TestSES::test_invalid_tags_send_email[test_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_lentest_invalid_name_len-test]": {
```

Comment:
This will also be printed in output logs. Consider using [param ID](https://docs.pytest.org/en/7.1.x/example/parametrize.html#set-marks-or-test-id-for-individual-parametrized-test) to avoid this.